apps/src/code-studio/pd/application/teacher1920/Section1AboutYou.jsx

```diff
@@ -89,25 +94,83 @@ export default class Section1AboutYou extends LabeledFormComponent {
 
     return (
       <FormGroup>
-        {!nominated && (
-          <p>
-            Thanks for your interest in the Code.org Professional Learning
-            Program! This application should take 10 - 15 minutes to complete.
-            Fields marked with a <span style={{color: 'red'}}>*</span> are
-            required.
-          </p>
-        )}
         {nominated && (
           <p>
             Congratulations on your nomination for a scholarship to the Code.org
-            Professional Learning Program! This application should take 10 - 15
-            minutes to complete. Fields marked with a{' '}
-            <span style={{color: 'red'}}>*</span> are required. We will let your
-            local partner know that you’ve been nominated as they consider your
-            application for the regional scholarships or discounts they have
-            available.
+            Professional Learning Program! We will let your local partner know
+            that you’ve been nominated as they consider your application for the
+            regional scholarship or discounts they have available.
           </p>
         )}
+
+        <h3>What’s in this application and how long will it take?</h3>
+        <p>
+          This application should take 10 - 15 minutes to complete. Fields
+          marked with a <span style={{color: 'red'}}>*</span> are required. Here
+          are the sections you will be asked to fill out:
+        </p>
+        <ul>
+          <li>
+            <span style={styles.bold}>Section 1: About you</span>
+            &nbsp; (Your and your principal’s contact info)
```

Comment:
It looks like the principal's contact info is now in section 2

Reply:
Great catch.  I missed this change in the spec.